feelpp/feel/CMakeLists.txt

```diff
@@ -167,7 +167,7 @@ if ( GOOGLE_PERFTOOLS_FOUND )
 endif()
 if (PETSC_FOUND)
   target_compile_definitions( feelpp PUBLIC FEELPP_HAS_PETSC FEELPP_HAS_PETSC_H )
-  target_include_directories( feelpp PUBLIC ${PETSC_PACKAGE_INCLUDES} ${PETSC_INCLUDE_DIR} PRIVATE ${PETSC_INCLUDE_CONF})
+  target_include_directories( feelpp PUBLIC ${PETSC_PACKAGE_INCLUDES} ${PETSC_INCLUDE_DIR} ${PETSC_INCLUDE_CONF})
```

Comment:
why do you need to put public the petsc conf dir?

Reply:
Because otherwise anything but the lib won't compile. For the qs, right now, only `${PETSC_DIR}/include` is included, whereas it should be `${PETSC_DIR}/include` + `${PETSC_DIR}/${PETSC_ARCH}include`.

Reply:
I thought it was redundant with PETSC_INCLUDE_DIR but Ok I understand